api/apps/api/test/project-scenarios.e2e-spec.ts

```diff
@@ -126,59 +128,107 @@ describe('ScenariosModule (e2e)', () => {
     await fixtures.GivenScenarioWasCreated();
     await fixtures.GivenContributorWasAddedToScenario();
     const response = await fixtures.WhenAcquiringLockForScenarioAsContributor();
-    fixtures.ThenScenarioLockInfoIsReturned(response);
+    fixtures.ThenScenarioLockInfoForContributorIsReturned(response);
   });
 
-  it('Owner acquires a lock for a scenario', async () => {
+  it('Owner acquires lock for a scenario', async () => {
     await fixtures.GivenScenarioWasCreated();
     const response = await fixtures.WhenAcquiringLockForScenarioAsOwner();
-    fixtures.ThenScenarioLockInfoIsReturned(response);
+    fixtures.ThenScenarioLockInfoForOwnerIsReturned(response);
   });
 
-  it('Fails to acquire a lock for a scenario as there was one already', async () => {
+  it('Fails to acquire lock for a scenario as there was one already', async () => {
     await fixtures.GivenScenarioWasCreated();
     await fixtures.GivenContributorWasAddedToScenario();
 
     let response = await fixtures.WhenAcquiringLockForScenarioAsOwner();
-    fixtures.ThenScenarioLockInfoIsReturned(response);
+    fixtures.ThenScenarioLockInfoForOwnerIsReturned(response);
 
     response = await fixtures.WhenAcquiringLockForScenarioAsContributor();
     fixtures.ThenScenarioIsLockedIsReturned(response);
   });
 
-  it('Fails to update a scenario as there was a lock in place by a different user', async () => {
+  it('Fails to update scenario as there was a lock in place by a different user', async () => {
     await fixtures.GivenScenarioWasCreated();
     await fixtures.GivenContributorWasAddedToScenario();
 
     let response = await fixtures.WhenAcquiringLockForScenarioAsOwner();
-    fixtures.ThenScenarioLockInfoIsReturned(response);
+    fixtures.ThenScenarioLockInfoForOwnerIsReturned(response);
 
     response = await fixtures.WhenUpdatingScenarioAsContributor();
     fixtures.ThenScenarioIsLockedIsReturned(response);
   });
 
-  it('Updates a scenario correctly as lock is in place by same user', async () => {
+  it('Updates scenario correctly as lock is in place by same user', async () => {
     await fixtures.GivenScenarioWasCreated();
     await fixtures.GivenContributorWasAddedToScenario();
 
     let response = await fixtures.WhenAcquiringLockForScenarioAsOwner();
-    fixtures.ThenScenarioLockInfoIsReturned(response);
+    fixtures.ThenScenarioLockInfoForOwnerIsReturned(response);
 
     response = await fixtures.WhenUpdatingScenarioAsOwner();
     fixtures.ThenScenarioIsUpdated(response);
   });
 
+  it('Releases scenario lock correctly', async () => {
+    await fixtures.GivenScenarioWasCreated();
+    const userToken = await fixtures.GivenUserIsLoggedIn('owner');
+
+    let response = await fixtures.WhenAcquiringLockForScenarioAsOwner();
+    fixtures.ThenScenarioLockInfoForOwnerIsReturned(response);
+
+    response = await fixtures.WhenReleasingLockForScenario(userToken);
+    fixtures.ThenNoContentIsReturned(response);
```

Comment:
minor, but I find this slightly misleading - what we are asserting here is really that the operation is successful, so maybe
```suggestion
    fixtures.ThenLockIsSuccessfullyReleased(response);
```

Reply:
same for similar cases below